src/components/notifications/AppExplorer/AppCard/index.tsx

```diff
@@ -32,16 +33,24 @@ const AppCard: React.FC<AppCardProps> = ({ name, description, logo, bgColor, url
     return specifiedMode === 'dark' ? bgColor.dark : bgColor.light
   }, [mode, bgColor])
 
+  const { activeSubscriptions } = useContext(W3iContext)
+
+  const subscribed = activeSubscriptions.some(element => element.metadata.name === name)
+
+  useEffect(() => {
+    if (ref.current) {
+      ref.current.style.setProperty('--local-bg-color', cardBgColor)
```

Comment:
Wouldn't this keep setting the property ?
Should add a check by getting the property first ?